disnake/ext/commands/bot_base.py

```diff
@@ -43,22 +43,10 @@
 from .view import StringView
 
 if TYPE_CHECKING:
-
-    from typing_extensions import ParamSpec
-
-    from disnake.interactions import ApplicationCommandInteraction
     from disnake.message import Message
 
     from ._types import Check, CoroFunc
 
-    ApplicationCommandInteractionT = TypeVar(
-        "ApplicationCommandInteractionT", bound=ApplicationCommandInteraction, covariant=True
-    )
-    AnyMessageCommandInter = Any  # Union[ApplicationCommandInteraction, UserCommandInteraction]
-    AnyUserCommandInter = Any  # Union[ApplicationCommandInteraction, UserCommandInteraction]
-
-    P = ParamSpec("P")
-
```

Comment:
Same as above, why delete these?

Reply:
I think they might be used by slash introspection as they need to exist in the globals.